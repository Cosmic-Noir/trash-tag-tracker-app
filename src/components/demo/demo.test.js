import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import Demo from "./demo";
import { BrowserRouter } from "react-router-dom";

describe("Demo Component", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDom.render(
      <BrowserRouter>
        <Demo />
      </BrowserRouter>,
      div
    );
    ReactDom.unmountComponentAtNode(div);
  });
  it("Renders the UI as expected", () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Demo />
      </BrowserRouter>
    ).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
