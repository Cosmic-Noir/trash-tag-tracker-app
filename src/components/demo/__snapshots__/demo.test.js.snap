// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo Component Renders the UI as expected 1`] = `[Function]`;
